openrtb-model/src/main/scala/com/powerspace/openrtb/conversion/ToNative.scala

```diff
@@ -0,0 +1,46 @@
+package com.powerspace.openrtb.conversion
+
+import com.google.openrtb.BidRequest
+import com.google.openrtb.BidRequest.Imp
+import com.google.openrtb.BidRequest.Imp.Native
+import com.google.openrtb.BidRequest.Imp.Native.RequestOneof
+import monocle.function.Each
+import monocle.function.Each.fromTraverse
+import scalaz.{Applicative, Traverse}
+
+object SeqTraverse {
+
+  implicit val seqTraverse = new Traverse[Seq] {
+    def traverseImpl[F[_], A, B](l: Seq[A])(f: A => F[B])(implicit F: Applicative[F]) = {
```

Comment:
why though? Isn't it already implemented in scalaz or cats?

Reply:
It is not. and it is scalaz code, since monocle relies on scalaz. I looked for it and didn't find it.